buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/shared-configuration.kt

```diff
@@ -125,15 +135,6 @@ fun Project.integrationTestUsesToolingApiFatJar() {
 }
 
 
-fun Project.integrationTestUsesToolingApiJar() {
```

Comment:
Can we also get rid of `integrationTestUsesToolingApiFatJar()` above and use "libsRepository.required = true" instead (in the _plugin-development_ project, which seems to be the only place where that is used)